bluebutton-data-pipeline-rif/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/model/BeneficiaryRow.java

```diff
@@ -279,29 +278,32 @@ public String toString() {
 		BENE_PTB_TRMNTN_CD,
 
 		/**
-		 * Type: <code>CHAR</code>, max chars: 12. See <a href=
+		 * TODO - need data dictionary link for BENE_CRNT_HIC_NUM Type:
+		 * <code>CHAR</code>, max chars: 12. See <a href=
 		 * "https://www.ccwdata.org/cs/groups/public/documents/datadictionary/???.txt">
 		 * CCW Data Dictionary: BENE_CRNT_HIC_NUM</a>.
 		 */
 		BENE_CRNT_HIC_NUM,
 
 		/**
-		 * Type: <code>CHAR</code>, max chars: 24. See <a href=
+		 * TODO - need data dictionary link for BENE_SRNM_NAME Type:
+		 * <code>CHAR</code>, max chars: 24. See <a href=
 		 * "https://www.ccwdata.org/cs/groups/public/documents/datadictionary/???.txt">
 		 * CCW Data Dictionary: BENE_SRNM_NAME</a>.
 		 */
 		BENE_SRNM_NAME,
 
 		/**
-		 * Type: <code>CHAR</code>, max chars: 5. See <a href=
+		 * TODO - need data dictionary link for BENE_GVN_NAME Type:
+		 * <code>CHAR</code>, max chars: 5. See <a href=
 		 * "https://www.ccwdata.org/cs/groups/public/documents/datadictionary/???.txt">
 		 * CCW Data Dictionary: BENE_GVN_NAME</a>.
 		 */
 		BENE_GVN_NAME,
 
 		/**
-		 * Type: <code>CHAR</code>, max chars: 1 <code>Optional</code>. See
-		 * <a href=
+		 * TODO - need data dictionary link for BENE_MDL_NAME Type:
+		 * <code>CHAR</code>, max chars: 1 <code>Optional</code>. See <a href=
 		 * "https://www.ccwdata.org/cs/groups/public/documents/datadictionary/???.txt">
```

Comment:
FWIW, I suspect that docs for these three fields doesn't exist, but you might double-check that with Sarah. If it doesn't exist, please remove the `TODO`s. Replace it with a comment along the lines of "This is the beneficiary's [first/middle/last] name."